stix_shifter_modules/sentinelone/README.md

```diff
@@ -0,0 +1,679 @@
+```
+
+#### STIX Translate results - output
+```json
+{
+    "type": "bundle",
+    "id": "bundle--245724b9-0860-4099-b24c-0dd298eeff4a",
+    "objects": [
+        {
+            "type": "identity",
+            "id": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "name": "sentinelone",
+            "identity_class": "events"
+        },
+        {
+            "id": "observed-data--08187b26-000a-4ce5-8da6-37b388bdbbf2",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2022-04-05T08:10:53.159Z",
+            "modified": "2022-04-05T08:10:53.159Z",
+            "objects": {
+                "0": {
+                    "type": "x-oca-event",
+                    "agent": "xx",
+                    "host_ref": "1",
+                    "created": "2022-04-05T07:52:02.051Z",
+                    "action": "IP Connect",
+                    "object_type": "ip",
+                    "process_ref": "5",
+                    "user_ref": "7"
+                },
+                "1": {
+                    "type": "x-oca-asset",
+                    "extensions": {
+                        "x-oca-endpoint": {
+                            "agent_uuid": "209e5033aedc4d50b3c056e0a01a9841",
+                            "machine_type": "xx",
+                            "endpoint_os": "windows"
+                        }
+                    },
+                    "ip_refs": ["2","4"],
+                    "hostname": "xx"
+                },
+                "2": {
+                    "type": "ipv4-addr",
+                    "value": "xx"
+                },
+                "3": {
+                    "type": "network-traffic",
+                    "dst_ref": "2",
+                    "dst_port": xx,
+                    "extensions": {
+                        "x-sentinelone-network-action": {
+                            "connection_status": "SUCCESS",
+                            "event_direction": "INCOMING"
+                        }
+                    },
+                    "protocols": [
+                        "null"
+                    ],
+                    "src_ref": "4",
+                    "src_port": xx
+                },
+                "4": {
+                    "type": "ipv4-addr",
+                    "value": "xx"
+                },
+                "5": {
+                    "type": "process",
+                    "command_line": "C:\\Windows\\System32\\xx -k termsvcs",
+                    "extensions": {
+                        "x-sentinelone-process": {
+                            "image_path": "C:\\Windows\\system32\\xx",
+                            "sha1_image": "a1385ce20ad79f55df235effd9780c31442aa234",
```

Comment:
This should be a file:hashes.'SHA-1' object

Reply:
The changes has been done in From_stix and To_Stix map file.